lib/tbd/geo.rb

```diff
@@ -347,6 +347,7 @@ def properties(surface = nil, argh = {})
 
     surf[:conditioned] = surf.key?(:heating) || surf.key?(:cooling)
     surf[:space      ] = space
+    surf[:occupied   ] = space.partofTotalFloorArea
```

Comment:
Keeping track of _in/directly-conditioned_ spaces that are 'unoccupied' (i.e plenums).

Reply:
This seems like a good idea.  For actual plenums you can check [`isPlenum`](https://github.com/NREL/OpenStudio/blob/develop/src/model/ThermalZone.hpp#L438) or [ `canBePlenum`](https://github.com/NREL/OpenStudio/blob/develop/src/model/ThermalZone.hpp#L444C10-L444C21).  That doesn't help as much for attics or crawlspaces so I think what you have here covers everything.

Reply:
Thanks @macumber. In the first releases, we actually did rely on a "plenum" spacetype-based method (which in part relied on _isPlenum_). [Here](https://github.com/rd2/osut/blob/362e0ac34426987d5b49c9e16e9ed71779ccd061/lib/osut/utils.rb#L1850), I go through why relying on _plenum_ categorization is inadequate. The solution now rests on [OSut::unconditioned?](https://github.com/rd2/osut/blob/362e0ac34426987d5b49c9e16e9ed71779ccd061/lib/osut/utils.rb#L2052)(space), which calls [OSut::setpoint(space)](https://github.com/rd2/osut/blob/362e0ac34426987d5b49c9e16e9ed71779ccd061/lib/osut/utils.rb#L1961). The latter catches indirectly-conditioned spaces, so yes we're OK on that front. What was missing here was catching whether the space was occupied or not.